nautobot/core/settings_funcs.py

```diff
@@ -123,7 +123,9 @@ def setup_structlog_logging(
         return
 
     django_apps.append("django_structlog")
-    django_middleware.append("django_structlog.middlewares.RequestMiddleware")
+    # Insert the middleware in second to last place so it comes ahead of
+    # django_prometheus.middleware.PrometheusAfterMiddleware, which consumes the request
+    django_middleware.insert(-1, "django_structlog.middlewares.RequestMiddleware")
```

Comment:
I'm curious @Kircheneer, should we instead find the `PrometheusAfterMiddleware` first then ensure we are inserting it ahead of it? I know of several deployments that declare their own entire middleware stack.